core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -545,21 +545,25 @@ public function getObjectsInContainer($path)
 
 
     /**
-     * @param $path
+     * @param string $path
+     * @param bool $isNew
      *
      * @return array|bool
      */
-    public function getMetaData($path)
+    public function getMetaData($path, $isNew = false)
     {
         try {
             $data['path'] = $path;
             $data['timestamp'] = $this->filesystem->lastModified($path);
             $data['size'] = $this->filesystem->fileSize($path);
             $data['mimetype'] = $this->filesystem->mimeType($path);
             $data['type'] = $data['mimetype'] === 'directory' ? 'dir' : 'file';
-
         } catch (FilesystemException | UnableToRetrieveMetadata $e) {
-            $this->xpdo->log(modX::LOG_LEVEL_ERROR, $e->getMessage());
+            if (!$isNew) {
+                $this->xpdo->log(modX::LOG_LEVEL_ERROR, $e->getMessage());
+            } else {
+                $this->xpdo->log(modX::LOG_LEVEL_ERROR, "\r\n\t" . $this->xpdo->lexicon('file_info_metadata_new', ['file' => $path]));
```

Comment:
Why do yo create a new line and a tab indent here?

Reply:
It's just easier to read the part of the output that's most important IMO.